lizrd/core/init.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch.nn.init import trunc_normal_
+
+
+def get_init_weight(shape, fan_in, init_type="uniform", scale=1.0, dtype=torch.float32):
+    if init_type == "uniform":
```

Comment:
can we change to the actual name of the initialisation? Like Kaiming, Glorot or whatever this particular one is

Reply:
+1